src/components/Modeler.vue

```diff
@@ -949,7 +933,7 @@ export default {
       this.setShapeStacking(shape);
 
       shape.component.$emit('click');
-    });
+    }, this);
```

Comment:
Using an arrow function will bind the value of this implicitly, so I don't think you need to pass `this` as the 3rd argument (just like the event handler above).

Reply:
But won't this mean we pass undefined as the callbackScope? Will that work then with the internal call? I'll try it to see if it still works.

Reply:
Not including the argument should be the same as explicitly passing `undefined`.

Reply:
It seems to work!